From e3eb467acb981b86348ba9d48a537e4b438905a6 Mon Sep 17 00:00:00 2001
From: Sveinung Kvilhaugsvik <sveinung84@users.sourceforge.net>
Date: Tue, 27 Oct 2015 20:37:41 +0100
Subject: [PATCH 1/2] generate_packets.py: ignore argument position

Stop depending on the position of command line arguments. Not an issue at
the moment since only one argument currently exists.
---
 common/generate_packets.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/generate_packets.py b/common/generate_packets.py
index 3b33849..cc1731f 100755
--- a/common/generate_packets.py
+++ b/common/generate_packets.py
@@ -40,7 +40,7 @@ import re, string, os, sys
 lazy_overwrite=0
 
 def verbose(s):
-    if len(sys.argv)>1 and sys.argv[1]=="-v":
+    if "-v" in sys.argv:
         print(s)
         
 def prefix(prefix,str):
-- 
2.1.4

